packages/blade-old/src/molecules/Modal/Modal.native.js

```diff
@@ -118,6 +118,7 @@ const Modal = ({ children, variant, visible, onClose, onBackdropClick }) => {
       <Space margin={[0]}>
         <Flex justifyContent="flex-end">
           <RNModal
+            testID="bottomsheet-modal"
```

Comment:
Why not accept `testID` from props and do this? We're doing this for other components as well. Refer Button or Card
```suggestion
             {...automation(testID)}
```